src/Microsoft.Health.Fhir.Core/Configs/ImportTaskConfiguration.cs

```diff
@@ -16,6 +16,7 @@ public class ImportTaskConfiguration
         private const int DefaultSqlCleanResourceBatchSize = 1000;
         private const int DefaultSqlMaxRebuildIndexOperationConcurrentCount = 3;
         private const int DefaultSqlMaxDeleteDuplicateOperationConcurrentCount = 3;
+        private const int DefaultSqlMaxDatatableProcessConcurrentCount = 3;
```

Comment:
What was it before?

Reply:
previous it is sequence, it is 1.

Reply:
previous convert to sql format is sequence => 1

Reply:
What is the reason for this change?